src/test/java/core/basesyntax/ServiceTest.java

```diff
@@ -0,0 +1,144 @@
+package core.basesyntax;
+
+import core.basesyntax.fruitshop.model.Fruit;
+import core.basesyntax.fruitshop.model.OperationType;
+import core.basesyntax.fruitshop.model.TransactionDto;
+import core.basesyntax.fruitshop.service.FileReaderServiceImpl;
+import core.basesyntax.fruitshop.service.FileWriterServiceImpl;
+import core.basesyntax.fruitshop.service.FruitShopServiceImpl;
+import core.basesyntax.fruitshop.service.OperationStrategyImpl;
+import core.basesyntax.fruitshop.service.TransactionDtoServiceImpl;
+import core.basesyntax.fruitshop.service.operation.BalanceOperationHandler;
+import core.basesyntax.fruitshop.service.operation.OperationHandler;
+import core.basesyntax.fruitshop.service.operation.PurchaseOperationHandler;
+import core.basesyntax.fruitshop.service.operation.ReturnOperationHandler;
+import core.basesyntax.fruitshop.service.operation.SupplyOperationHandler;
+import core.basesyntax.fruitshop.storage.Storage;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ServiceTest {
+    private FileReaderServiceImpl readerService;
+    private FileWriterServiceImpl fileWriterService;
+    private FruitShopServiceImpl fruitShopService;
+    private TransactionDtoServiceImpl transactionDtoService;
+
+    @Before
+    public void beforeAll() {
+        Map<OperationType, OperationHandler> operationHandlerMap = new HashMap<>();
+        operationHandlerMap.put(OperationType.BALANCE, new BalanceOperationHandler());
+        operationHandlerMap.put(OperationType.SUPPLY, new SupplyOperationHandler());
+        operationHandlerMap.put(OperationType.PURCHASE, new PurchaseOperationHandler());
+        operationHandlerMap.put(OperationType.RETURN, new ReturnOperationHandler());
+        OperationStrategyImpl operationStrategy = new OperationStrategyImpl(operationHandlerMap);
```

Comment:
OperationStrategyImpl operationStrategy is never used

Reply:
deleted